client/index.tpl.html

```diff
@@ -58,6 +60,14 @@
         <script src="bower_components/cryptojs/lib/PBKDF2.js"></script>
         <script src="bower_components/cryptojs/lib/HMAC.js"></script>
         <script src="bower_components/cryptojs/lib/AES.js"></script>
+
+        <script src="bower_components/jquery-ui/jquery-ui.js"></script>
```

Comment:
it apears it doesn't, have a look at the build `index.html`, the core of jquery-ui is missing :/